scripts/ffetch.js

```diff
@@ -160,7 +160,7 @@ function assignOperations(generator, context) {
 }
 
 export default function ffetch(url) {
-  let chunkSize = 255;
+  let chunkSize = 500;
```

Comment:
fwiw, you should be able to modify this on a per call basis by doing something like:

`ffetch(...).chunks(100)`

not that big a deal if you just want to modify it globally for all calls as you are doing here.